client/src/test/java/org/eclipse/hono/client/CommandResponseTest.java

```diff
@@ -324,7 +324,7 @@ public void testForNoDeviceIdInReplyToId() {
     @Test
     public void testForDeviceAndTenantIds() {
         final CommandResponse response = CommandResponse.from(
-                Command.getRequestId(CORRELATION_ID, REPLY_TO_ID, DEVICE_ID),
+                Command.getRequestId(CORRELATION_ID, REPLY_TO_ID, DEVICE_ID, false),
```

Comment:
what about tests for the case where the flag is `true`?

Reply:
I've added tests for that.